Library/Homebrew/formula.rb

```diff
@@ -405,11 +413,19 @@ def linked_keg
     Pathname.new("#{HOMEBREW_LIBRARY}/LinkedKegs/#{name}")
   end
 
+  def latest_head_prefix
+    head_prefixes = installed_prefixes.select do |pn|
+      pn.basename.to_s.start_with?("HEAD")
```

Comment:
FYI, we can ignore `head_prefix_up_to_date` for now in this PR.  It should be part of #274, after we merge this PR.